pkg/internal/ebpf/httptracer/httptracer.go

```diff
@@ -22,6 +22,10 @@ import (
 //go:generate $BPF2GO -cc $BPF_CLANG -cflags $BPF_CFLAGS -target amd64,arm64 bpf ../../../../bpf/tc_http_tp.c -- -I../../../../bpf/headers
 //go:generate $BPF2GO -cc $BPF_CLANG -cflags $BPF_CFLAGS -target amd64,arm64 bpf_debug ../../../../bpf/tc_http_tp.c -- -I../../../../bpf/headers -DBPF_DEBUG
 
+const (
+	httpTracerTCHandle = 0xb311
```

Comment:
What are these constants and how were they picked? Maybe a comment would help?

Reply:
These are the tc handles - they need to be unique - cilium conveniently uses `1` for their TC attachment. If they collide, we end up replacing (or being replaced). I initially thought of iterating the loaded tc programs and computing a handle value that is not being used, but that is too complicated for what we get in exchange and it does not guarantee we won't be ejected by the next 3rd-party tc program being loaded - so I chose high numbers that are unlikely to be used, which all start with `0xbe1` - a pun on beyla naming itself.

I thought of aggregating these constants in a single shared file, but that has the downside of leaking implementation detail/breaking encapsulation for otherwise unrelated "modules". Maybe I should do that anyway and place them in ebpfcommon?

Reply:
Oh that's totally reasonable, it's fine for them to be as such. Can we just put a comment on each saying they must be unique and I think putting them in a common place with constants is a good idea. If we need to all one in the future. There's this Go syntax with iota + a number, you can use that, where the first number will be 0xb311.

Reply:
As discussed offline, these have been moved into `ebpfcommon`.